src/SearchResults.js

```diff
@@ -1,9 +1,21 @@
-import React from "react";
+import React, { useState } from "react";
 import moment from "moment";
 
+import "./App.css";
+
 const SearchResult = props => {
+  const [active, setActive] = useState(null);
+
+  const handleClick = index => {
+    active === index ? setActive(null) : setActive(index);
```

Comment:
This is a good attempt, but it looks like if you click on any row, then all of them become highlighted. The exercise asks you to make each row *independent*, meaning that multiple rows can be highlighted at once.

The idea is to think carefully about how we could split up our components so that they can have their own state. Can you work out how to do that?